src/pages/publication/PublicationFormValidationSchema.ts

```diff
@@ -0,0 +1,63 @@
+import * as Yup from 'yup';
+import { ReferenceType, JournalArticleType, BookType, DegreeType, ReportType } from '../../types/references.types';
+import { LanguageCodes } from '../../types/language.types';
+import { TFunction } from 'i18next';
+
+export const getPublicationValidationSchema = (t: TFunction) =>
+  Yup.object().shape({
+    entityDescription: Yup.object().shape({
+      mainTitle: Yup.string().required(t('feedback.required_field')),
```

Comment:
fremdeles ikke flere felter som er required ?

Reply:
`publicationType` hvertfall. Ellers ingen kjente. `isPublished` for hver fil er bool og blank by default, så den burde også vært required